spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -64,8 +64,17 @@
   describe 'destory' do
     context 'with a claim' do
       it 'on an exception gives an error and redirects' do
-        fake_login_known_user(@user)
+        fake_login_known_user(User.find(collection.all_owners.first.user_id))
+        collection.challenge = nil
+        collection.save
+        delete :destroy, id: claim.id, collection_id: collection.name
+        it_redirects_to_with_error(collection_path(collection), \
+                                   "What challenge did you want to work with?")
+      end
+
+      it 'on an exception gives an error and redirects' do
```

Comment:
This has the same description as the previous test -- what's the difference between the two of them? The one above it looks like `it "gives and error and redirects if the claim's collection has no challenge"` and this one looks like it's more generic, i.e. `it "gives and error and redirects if record can't be destroyed"`.